### DISCUSSION TITLE
Where is the best location to add code to touch parent folder upon file edit?

### DISCUSSION THREAD

**Discussion by ghost** [gsantner/markor#2399] [Q&A]
This is taken from issue https://github.com/gsantner/markor/issues/2382

1. Where would be the best location to touch a prent folder after a text file is edited?

2. And how can I do it? (What is the relevant command?)

**Comment by harshad1** [comment]
1. Probably in the Document class.
2. Chatgpt is your friend
3. I still oppose merging this into master. But you can always have your own fork.
4.

**Comment by ghost** [comment]
Thanks @harshad1. I have now solved it as in this commit: https://github.com/gsantner/markor/issues/2382#issuecomment-2322975172
